spec/models/piece_spec.rb

```diff
@@ -1,11 +1,41 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
-  before do
+  before :each do
     # Note: We want the board to be emptry for this test.
     allow_any_instance_of(Game).to receive(:populate_board!).and_return true
   end
 
+  describe '#move_to!' do
+    before :each do
+      @piece_white = FactoryGirl.create(:piece, row: 0, column: 0, color: 'white')
+      @piece_black = FactoryGirl.create(:piece, row: 0, column: 7, color: 'black')
+    end
```

Comment:
instead of using a before block. you can use `let`

```
let(:piece_black) { FactoryGirl.create(:piece, row: 0, column: 7, color: 'black') }
```

Then `piece_black` will be available in all of your tests.